packages/build/src/models/documentGeneratorOptions.ts

```diff
@@ -0,0 +1,4 @@
+export interface IDocumentGeneratorOptions {
+  specs?: string[];
```

Comment:
I've set the built-in enum as a union type, it might help us to know more about the property.

```ts
export enum DocumentGeneratorSpec {
  oas3 = 'oas3',
}

export interface IDocumentGeneratorOptions {
  specs?: (string | DocumentGeneratorSpec)[];
  folderPath: string;
}

```